src/providers/speak/speak.test.ts

```
@@ -0,0 +1,142 @@
+    expect(getUploadTuning(10 * MB + 1)).toEqual({ chunkSize: 10 * MB, parallelUploads: 3 });
+    expect(getUploadTuning(500 * MB)).toEqual({ chunkSize: 10 * MB, parallelUploads: 3 });
+  });
+
+  it('uses 20 MB x 3 parallel above 500 MB', () => {
+    expect(getUploadTuning(500 * MB + 1)).toEqual({ chunkSize: 20 * MB, parallelUploads: 3 });
+  });
+});
+
+describe('uploadVideoEmbed', () => {
+  const media = (size: number) =>
+    ({ path: 'file:///tmp/video.mp4', size, filename: 'video.mp4' }) as any;
```

ESLint

[error] Replace `)·as·any` with `·as·any)`

(prettier/prettier)